app/Helpers/database/game.php

```diff
@@ -993,17 +994,10 @@ function GetPatchData(int $gameID, ?User $user, int $flag): array
         'ConsoleID' => $game->ConsoleID,
         'ImageIconURL' => media_asset($game->ImageIcon),
         'Achievements' => [],
+        'AchievementSets' => [],
```

Comment:
I'd rather leave this out until we actually populate it. The name might change depending on how the data is returned. For backwards compatibility we want to keep the core set in the Achievements list, and the additional sets would be in here. This name has some implications of holding both.

Additionally, there could be leaderboards for the subset, so it won't just be achievements in here.

Reply:
That makes sense. Removed `AchievementSets` from the response in latest.